### PR TITLE
Power all methods off CSV files

### PR DESCRIPTION
**Pull request by brideo**
https://meanbee.atlassian.net/browse/MB-35

### CODE REVIEW

File: app/code/community/Meanbee/Royalmail/Model/Shipping/Carrier/Royalmail/Firstclasslettersignedfor.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+
+class Meanbee_Royalmail_Model_Shipping_Carrier_Royalmail_Firstclasslettersignedfor
+    extends Meanbee_Royalmail_Model_Shipping_Carrier_Royalmail_Abstract {
+
+    protected function getRates() {
+        $helper = Mage::helper('royalmail');
+        $rates = $this->_loadCsv('firstclass_letter_signedfor');
+
+        if ($helper->getWorldZone($this->_getCountry()) == 'gb') {
```
Comment (punkstar) [inline app/code/community/Meanbee/Royalmail/Model/Shipping/Carrier/Royalmail/Firstclasslettersignedfor.php]: Would be good to use constants for GB, e.g. `Meanbee_Royalmail_Helper_Data::WORLD_ZONE_GB`.

**Comment by bobbyshaw** [comment]
Is this ok to be merged and the extension re-packaged?

**Comment by brideo** [comment]
I’m still waiting on a code review for that one, would you like me to chase it up in the morning?

File: app/code/community/Meanbee/Royalmail/Model/Shipping/Carrier/Royalmail/Specialdeliverynextdaysaturday.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ */
+
+class Meanbee_Royalmail_Model_Shipping_Carrier_Royalmail_Specialdeliverynextdaysaturday
+    extends Meanbee_Royalmail_Model_Shipping_Carrier_Royalmail_Abstract {
+
+    protected function getRates() {
+        $helper = Mage::helper('royalmail');
+        $rates = $this->_getRatesCsv();
+
+        if ($helper->getWorldZone($this->_getCountry()) == 'gb') {
```
Comment (punkstar) [inline app/code/community/Meanbee/Royalmail/Model/Shipping/Carrier/Royalmail/Specialdeliverynextdaysaturday.php]: Should be a constant for GB.

**Comment by punkstar** [comment]
Cool, this looks good.  My only beef is that insurance charges are still hardcoded in the CSV files.  We could probably better handle that by either adding configuration values in the admin to define the amount of insurance for each method, or power them off different CSVs completely.  I'll open a new ticket for that so we can close this down.

Feel free to merge when that one constant issue is fixed! :+1:

**Pull request merged**